mkinitcpio: add page (#5250)
tldr-pages/tldr: pages/linux/mkinitcpio.md

# mkinitcpio

> > Bash script for create an initial ramdisk environment.

Reviewer: ```suggestion
> Generates one more initial ramdisk environments for booting the Linux kernel based on the specified preset(s).
```

> > Generates one more initial ramdisk environments for booting the Linux kernel based on the specified preset(s).

Reviewer 2: Perhaps this will read better?
```suggestion
> Generates initial ramdisk environments for booting the Linux kernel based on the specified preset(s).
```

> More information: <https://man.archlinux.org/man/mkinitcpio.8>.

> - Do a dry run and display the output:

Reviewer: This would be better for a new user:
```suggestion
- Perform a dry run (print what would be done without actually doing it):
```

`mkinitcpio`

> - Generate the preset provided by `linux` package:

Reviewer: ```suggestion
- Generate a ramdisk environment based on the `linux` preset:
```

`mkinitcpio --preset {{linux}}`

> - Generate the preset provided by `linux-lts` package:

Reviewer: ```suggestion
- Generate a ramdisk environment based on the `linux-lts` preset:
```

`mkinitcpio --preset {{linux-lts}}`

> - Generate all existing presets (used to regenerate all the initramfs images after a change in `/etc/mkinitcpio.conf`):

Reviewer: ```suggestion
- Generate ramdisk environments based on all existing presets (used to regenerate all the initramfs images after a change in `/etc/mkinitcpio.conf`):
```

`mkinitcpio --allpresets`

> - Generate an image using an alternative configuration file:

Reviewer: ```suggestion
- Generate an initramfs image using an alternative configuration file:
```

`mkinitcpio --config {{path/to/mkinitcpio.conf}} --generate {{path/to/initramfs.img}}`

> - Generate an image for a kernel other than the one currently running (the installed kernel releases can be found in `/usr/lib/modules/`):

Reviewer: ```suggestion
- Generate an initramfs image for a kernel other than the one currently running (the installed kernel releases can be found in `/usr/lib/modules/`):
```
